src/Utils/StringHelper.php

```diff
@@ -20,8 +20,8 @@ public static function ensureMaxLength(?string $string, int $length): ?string
             return null;
         }
 
-        if (mb_strlen($string) > $length) {
-            $string = mb_substr($string, 0, $length);
+        if (\mb_strlen($string) > $length) {
```

Comment:
The codestyles are purely managed by php-cs-fixer.
The build will fail if they are broken, no need for manual adjustments.
You can use `composer codestyle-fix` to apply them.

Reply:
No, i had to adjust it manually because this command didn't do it!

Reply:
There is a link somewhere in the php-cs-fixer rule, that leads to a place where the explanation is better than I could ever repeat, but here is a link to start with: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/3048

Reply:
ah ok, quoting https://github.com/symfony/symfony/pull/32717#issuecomment-514664927: Only some functions can be optimized but not all. Learned something new ;)